wm-system/main/src/main/kotlin/com/ajaxproject/warehouse/service/WaybillServiceImpl.kt

```diff
@@ -29,7 +29,7 @@ import java.time.ZoneOffset
 @Suppress("TooManyFunctions")
 class WaybillServiceImpl(
```

Comment:
Please, fix 
1. class package
2. avoid dependency on infrastructure

Reply:
But I have already fixed dependency on infrastructure and I didn't break up all project to DDD, only product module, so class is placed as it was before refactoring

Reply:
discussed offline